phpmyfaq/inc/Bootstrap.php

```diff
@@ -259,7 +259,7 @@
  * @param mixed  $context  It optionally contains an array of every variable
  *                         that existed in the scope the error was triggered in.
  *
- * @return bool
+ * @return boolean|null
```

Comment:
I would like to scrutinize the *Scrutinizer*.

This function either terminates using `exit()` or returns `true`. It might as well return nothing.
It appears to be used in nothing but **_[`set_error_handler()`](https://github.com/thorsten/phpMyFAQ/blob/2.9/phpmyfaq/inc/Bootstrap.php#L131)_**.

If you agree I'll add it to a PR of updates I'm working on. I'm going to upgrade from 2.8.15 to the latest 2.9.0 RC2. I have been wanting to resolve https://github.com/thorsten/phpMyFAQ/issues/1044 so upgrading will allow me to test  https://github.com/thorsten/phpMyFAQ/commit/7f76d1e5cc1d3e11f42b666058c8c3a7fc009abd however, I did not want to upgrade to further 2.8._x_ versions without my SMTP code (already added to 2.9._x_) and quite a few other mods I have in the works. I also did not want to use 2.9 until it seemed fairly stable.

Before you decide to release 2.9.0, make sure you give a few days notice on your forum, or GitHub somehow, as I'd like to ensure I can test and/or mention a few things before the database design is locked till 2.10.0